test/integration_find_test.go

```diff
@@ -0,0 +1,88 @@
+package test_test
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/gruntwork-io/terragrunt/test/helpers"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	testFixtureFindBasic  = "fixtures/find/basic"
+	testFixtureFindHidden = "fixtures/find/hidden"
+)
+
+func TestFindBasic(t *testing.T) {
+	t.Parallel()
+
+	helpers.CleanupTerraformFolder(t, testFixtureFindBasic)
+
+	stdout := &bytes.Buffer{}
+	stderr := &bytes.Buffer{}
+
+	err := helpers.RunTerragruntCommand(t, "terragrunt find --experiment cli-redesign --no-color --working-dir "+testFixtureFindBasic, stdout, stderr)
```

Comment:
why not to use `RunTerragruntCommandWithOutput`? it returns sting values for output and err

Reply:
Updated. Why do we have both in the test suite? Should we remove one?